common/Strings/en-us/Resources.resw

```diff
@@ -205,12 +205,31 @@
     <value>Uptime:</value>
     <comment>Title for the uptime property of a compute system. (The amount of time its been running)</comment>
   </data>
+  <data name="HyperVAdminAddUser" xml:space="preserve">
+    <value>Add User to Hyper-V admin group</value>
+  </data>
+  <data name="HyperVErrorTitle" xml:space="preserve">
+    <value>Microsoft Hyper-V</value>

```

Comment:
Do we have to lock any of the words in these strings? Like making sure Hyper-V doesn't get translated?

Reply:
Changed.